app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_comment_count
+
+  def update_comment_count
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [Optional] Good job implementing the requested changes, after you should make the `update_comment_count` method private 🙃 . Kindly do the same for `update_like_count` and `update_posts_count` methods 😁